aioca/_catools.py

```diff
@@ -27,6 +28,7 @@
 from .types import AugmentedValue, Count, Datatype, Dbe, Format, Timeout
 
 T = TypeVar("T")
+PVS = Union[List[str], Tuple[str]]
```

Comment:
```suggestion
PVS = Union[List[str], Tuple[str, ...]]
```
Seems to be the correct form of a variable length tuple of strings: https://docs.python.org/3/library/typing.html#typing.Tuple

Reply:
I had missed this distinction.